src/main/resources/freemarker/dac-expired.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Broad Data Use Oversight System - Researcher - Data Access Request Expired</title>
+</head>
+
+<body style="text-align: center; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; width: 100%; height: 100%; margin: 0; padding: 0;">
+<div style="text-align: left; margin: 0 auto; width: 600px;">
+  <table style="width: 600px">
+    <tr>
+      <td style="padding: 15px 15px 0; text-align: center;">
+        <img src="https://duos.org/images/favicon/android-chrome-192x192.png" height="100px" width="100px" alt="DUOS Logo">
+      </td>
+    </tr>
+    <tr>
+      <td id="userName" style="padding: 15px 15px 0; font-family: 'Montserrat', sans-serif; font-size: 22px; color: #1F3B50; text-align: left; line-height: 25px; display: block; font-weight: 500;">
+        Dear ${researcherName},
+      </td>
+    </tr>
+    <tr>
+      <td style="padding: 15px 15px 0; font-family: 'Montserrat', sans-serif; font-size: 16px; color: #1F3B50; text-align: justify; line-height: 25px;">
+        <p id="content" style="margin: 0;">
+          Your Data Access Request ${darCode} has expired. Please complete a progress report to preserve your access to this data or close out your project.
```

Comment:
When we have a page for the progress report, it would be nice to update this email to contain a link to it.